src/App.js

```diff
@@ -19,6 +20,7 @@ export default function App() {
           <Route path="*" component={AppBar} />
         </Switch>
         <Route exact path="/" component={Home} />
+        <Route exact path="/allposts" component={AllPosts} />
```

Comment:
If this all posts page is going to be the root having both of these makes no sense